src/core/src/service_interfaces/StatusHandler.py

```diff
@@ -776,10 +796,206 @@ def __try_add_error(error_list, detail):
     def __set_errors_json(self, error_count_by_operation, errors_by_operation):
         """ Compose the error object json to be added in 'errors' in given operation's summary """
         message = "{0} error/s reported.".format(error_count_by_operation)
-        message += " The latest {0} error/s are shared in detail. To view all errors, review this log file on the machine: {1}".format(len(errors_by_operation), self.__log_file_path) if error_count_by_operation > 0 else ""
+        message += " The latest {0} error/s are shared in detail. To view all errors, review this log file on the machine: {1}".format(
+            len(errors_by_operation), self.__log_file_path) if error_count_by_operation > 0 else ""
         return {
             "code": Constants.PatchOperationTopLevelErrorCode.SUCCESS if error_count_by_operation == 0 else Constants.PatchOperationTopLevelErrorCode.ERROR,
             "details": errors_by_operation,
             "message": message
         }
+
+    def __set_error_detail(self, error_code, message):
+        formatted_message = self.__ensure_error_message_restriction_compliance(message)
+        return {
+            "code": str(error_code),
+            "message": str(formatted_message)
+        }
+    # endregion
+
+    # region - Patch Truncation
+    def get_truncated_patches(self):
+        return self.__truncated_patches
```

Comment:
General comment: Variable and function naming needs consistency with existing usage.